lib/broccoli/angular2-app.js

```diff
@@ -23,7 +26,7 @@ class Angular2App extends BroccoliPlugin {
     options = options || {};
 
     this._options = options;
-    this._sourceDir = options.sourceDir || 'src/client';
+    this._sourceDir = ngConfig.defaults.sourceDir || 'src';
```

Comment:
Please keep the options if it's passed in. basically `options.sourceDir || ngConfig.defaults.sourceDir`.  The config itself should have a default value. The `|| 'src'` can be kept for this PR, but we need the options for Material (we have 2 apps).